Inventory Management System/Product.cs

```diff
@@ -1,12 +1,88 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
+using System.Xml.Linq;
+using ConsoleTables;
 
 namespace Inventory_Management_System
 {
     internal class Product
     {
+        public string Name { get; set; }
+        public double Price { get; set; }
+        public int Quantity { get; set; }
+        public Product()
+        {
+            this.Name = null;
+            this.Price = 0;
+            this.Quantity = 0;
+        }
```

Comment:
please remove this constructor, since the default value of string is null, double is 0, and for int is 0.